docs/conf/portland/2022/news/conference-going-virtual.rst

```diff
@@ -0,0 +1,98 @@
+:template: {{year}}/generic.html
+
+.. post:: Jan 26, 2022
+   :tags: {{shortcode}}-{{year}}, virtual
+
+Conference will be only virtual
+===============================
+
+We won't bury the lede on the announcement today:
+
+   The Portland 2022 conference held on May 23-25 will be fully virtual.
+
+We make this decision with the understanding that it will disappoint some of you.
+We also make this decision with the knowledge that it will relieve some of you from anxiety.
+I will explain a bit more below the reasoning for this decision,
```

Comment:
Do we want to keep the "we" voice or make this blog kind of a "personal message from the conference chair"? It could work either way but if we want to use "I" in the post, we should spell our Eric's name in the beginning so that the tone is consistent.

Reply:
Hrm yea... I'm terrible with tone. I think `We` is probably best.